toolbox/Dockerfile

```diff
@@ -11,12 +10,12 @@ RUN wget https://github.com/rackspace-infrastructure-automation/tfenv/archive/v0
   && ln -s /var/opt/tfenv-0.6.0/bin/* /usr/local/bin
 RUN pip install --upgrade pip && pip install --progress-bar=off awscli
 
-# tuvok and json2hcl, then run tuvok once to be sure it loads correctly
-ADD https://github.com/kvz/json2hcl/releases/download/v${JSON2HCL_VERSION}/json2hcl_v${JSON2HCL_VERSION}_linux_amd64 /usr/local/bin/json2hcl
-RUN chmod +x /usr/local/bin/json2hcl
+# install tuvok
 RUN pip install git+https://github.com/rackerlabs/tuvok.git@$TUVOK_VERSION
 
 # recent terraform versions
+RUN tfenv install 0.12.20
+RUN tfenv install 0.12.17
```

Comment:
You might want to change the bit a few lines down to set the default version?

```Dockerfile
RUN tfenv use 0.12.17
```

...Then again, you might not want to. 🙂

Reply:
My initial thought was to leave the default version at 0.11.x.  Once we start converting customers over, I think it will be worth evaluating.  But happy to adjust as folks see fit.

Reply:
I think that is a good thought.